src/sentry/static/sentry/app/components/dataPrivacyRules/eventIdFieldStatusIcon.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import ControlState from 'app/views/settings/components/forms/field/controlState';
+import {t} from 'app/locale';
+import Tooltip from 'app/components/tooltip';
+import {IconClose, IconCheckmark} from 'app/icons';
+
+import {EventIdStatus} from './types';
+
+type Props = {
+  onClickIconClose: () => void;
+  status?: EventIdStatus;
+};
+
+const EventIdFieldStatusIcon = ({status, onClickIconClose}: Props) => {
+  switch (status) {
+    case EventIdStatus.ERROR:
+    case EventIdStatus.INVALID:
+    case EventIdStatus.NOT_FOUND:
+      return (
+        <CloseIcon onClick={onClickIconClose}>
+          <Tooltip title={t('Clear Event ID')}>
+            <IconClose color="red" />
+          </Tooltip>
+        </CloseIcon>
+      );
+    case EventIdStatus.LOADING:
+      return <ControlState isSaving />;
+    case EventIdStatus.LOADED:
+      return <IconCheckmark color="green" />;
```

Comment:
Should we use a color from the theme instead?

Reply:
it is from the theme 😉 

![image](https://user-images.githubusercontent.com/29228205/82920154-38bc2380-9f77-11ea-937f-5fbe810be7f4.png)